.github/workflows/ci.yml

```diff
@@ -146,6 +146,27 @@ jobs:
         chmod a+x builder
         ./builder build -p ${{ env.PACKAGE_NAME }}
 
+  openbsd:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build ${{ env.PACKAGE_NAME }} + consumers
+        uses: cross-platform-actions/action@v0.10.0
+        with:
+          operating_system: openbsd
+          architecture: x86-64
+          version: '7.2'
+          shell: bash
+          run: |
+            sudo pkg_add git
+            sudo pkg_add py3-urllib3
+            pkg_info
+            sysctl -n kern.version
+            sudo bash -c "echo \"permit keepenv nopass :wheel as root\" > /etc/doas.conf"
+            python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz', 'builder')"
```

Comment:
can we get aws-crt-builder doing any of this boilerplate?

I guess we're using urllib to acquire the builder... but maybe there's a different shell cmd we can invoke to do the download?

whatever makes it simplest to copy/paste this into other repo's ci.yml files... there's always a certain amount of copy/paste. it's always a balance

Reply:
Yes! Sort of.

- git: now installed by builder (v0.9.35)
- The pkg_info and sysctl lines: neither are necessary, only for data which might be helpful during troubleshooting. I've removed both lines.
- The `sudo bash` line: Removed as there's no dependency on doas any more (when using builder v0.9.35+).

Using urllib is interesting. While there is at least one command line util in the base OS install we could use to pull builder.pyz down, there's a weird challenge here with OpenBSD's package management tools. There is no mechanism to tell pkg_add "install the latest version of python". You have to give it a version. So it seems the choice is:

1. `pkg_add python%3.10` (for example) and carry the burden forward of bumping this version every time it becomes unavailable in the pkg repo, or
2. Install urllib3 which will pull in a valid version of python as a dependency. We could use the base command line tool to pull down builder.pyz in this case, but someone would likely come along and wonder why urllib is being installed. And I suppose if we continue to use urllib to pull builder.pyz, it looks consistent with the other jobs.

Please give the diff in my latest push a look and let me know what you think.